tests/codegen/snapshots/typescript/variables.ts

```diff
@@ -4,6 +4,7 @@ type OperationNameResult = {
 
 type PersonInput = {
     name: string
+    age: number | undefined
```

Comment:
should we also do `null` here?

Reply:
My typescript isn't that great, but I think that the intent is that `personInput.age` would be unset (`undefined`) in the request payload (rather than having it be `null`).  My typescript isn't that great -- but at least `JS` has a better way to express that "this key was never set" (`undefined`) vs. "this key was set to some default 'missing' value" (`null`).  I think that `UNSET` most closely maps to the "this key was never set" semantics -- so `undefined` seems like the right choice to me here...

Reply:
you're right 😊